config_module/builders/builders.json

```diff
@@ -55,6 +55,14 @@
                 },
                 "virtual_builder_tags":["coverage"]
             }
+        },
+        "nodeprecated":{
+            "properties":{
+                "cmake_defs":{
+                    "CMAKE_CXX_FLAGS": "-Wno-deprecated-declarations"
+                },
+                "virtual_builder_tags":["nodeprecated"]
```

Comment:
we don't want to add another tag for that but rather "hide" it. (not a important dividing feature across builders)